ml/src/test/resources/sms_pipeline_deployment/sms_pipeline.model/stages/4_mojoModel_4886dd97b3a0/metadata/part-00000

```diff
@@ -1 +1 @@
-{"class":"org.apache.spark.ml.h2o.models.H2OMOJOModel","timestamp":1513945002816,"sparkVersion":"2.2.1","uid":"mojoModel_4886dd97b3a0","paramMap":{"predictionCol":"label","outputCol":"prediction_output","featuresCols":["tf_idf"]}}
+{"class":"org.apache.spark.ml.h2o.models.H2OMOJOModel","timestamp":1513945002816,"sparkVersion":"2.2.1","uid":"mojoModel_4886dd97b3a0","paramMap":{"labelCol":"label","outputCol":"prediction_output","featuresCols":["tf_idf"]}}
```

Comment:
@jakubhava Shouldn't we also support backward compatibility of MOJO files when we rename parameters?

Reply:
Good point, yeah. At this point I would support the backwards compatibility between in between minor releases. New Major release should be able to break this ( and allow us to clean). WDYT?

Reply:
Yeah, it makes sense. Should we also remove the deprecated properties at the same time?

Reply:
I'm thinking about solving the backwards in a different PR. WDYT?

Reply:
yup, let's make a new PR